core/templates/pages/exploration-player-page/layout-directives/progress-nav.component.html

```diff
@@ -0,0 +1,112 @@
+<div class="progress-nav-bar row space-between center">
+  <!-- Wrapping div on first button keeps the second button right-aligned
+    even when the first button is not present. -->
+  <div class="row-item">
+    <!-- ng-if on this second wrapping div because ng-if lags on buttons. -->
+    <div *ngIf="hasPrevious && !helpCardHasContinueButton">
+      <button mat-raised-button
+              aria-label="Previous Card"
+              (click)="changeCard(displayedCardIndex-1)"
+              class="protractor-test-back-button oppia-back-button oppia-nav-button oppia-nav-left-button">
+        <i class="material-icons oppia-vcenter" aria-hidden="true">&#xE5C4;</i>
+      </button>
+    </div>
+  </div>
+  <div *ngIf="hasNext" class="row-item align-right">
+    <button mat-raised-button
+            (click)="changeCard(displayedCardIndex+1)"
+            aria-label="Next Card"
+            class="oppia-next-button protractor-test-next-button oppia-nav-button">
+      <i class="material-icons oppia-vcenter" aria-hidden="true">&#xE5C8;</i>
+    </button>
+  </div>
+  <div *ngIf="!hasNext">
+    <div *ngIf="submitButtonIsShown && !shouldContinueButtonBeShown() && !helpCardHasContinueButton" class="oppia-nav-button">
+      <!-- Special case for the 'Continue' interaction. -->
+      <button mat-button
+              *ngIf="interactionId === 'Continue'"
+              (click)="submit.emit()"
+              class="oppia-learner-confirm-button protractor-test-continue-button oppia-autofocus"
+              oppiaFocusOn="continue-btn">
+        {{ interactionCustomizationArgs.buttonText.value.unicode }}
+        <i class="fas fa-arrow-right oppia-arrow-right"></i>
+      </button>
+      <button mat-button
+              *ngIf="shouldGenericSubmitButtonBeShown()"
+              class="oppia-learner-confirm-button protractor-test-submit-answer-button"
+              (click)="submit.emit()"
+              [disabled]="submitButtonDisabled">
+        {{ 'I18N_INTERACTIONS_SUBMIT' | translate }}
+      </button>
+    </div>
+    <oppia-continue-button *ngIf="shouldContinueButtonBeShown()"
+                           [focusLabel]="CONTINUE_BUTTON_FOCUS_LABEL"
+                           [isLearnAgainButton]="isLearnAgainButton"
+                           (clickContinueButton)="clickContinueButton.emit()"
+                           class="d-block continue-button">
+    </oppia-continue-button>
+  </div>
+</div>
+
+<style>
```

Comment:
Why are these stylings different from the original stylings?

Reply:
`ui-bootstrap` (bootstrap for angularjs) and `ng-bootstrap` (bootstrap for angular) has some differences.
So, I have to add styles to make the component look as close as possible to the original directive.